crates/testing/src/task_helpers.rs

```diff
@@ -171,7 +171,19 @@ pub fn key_pair_for_id(node_id: u64) -> (<BLSPubKey as SignatureKey>::PrivateKey
     (private_key, public_key)
 }
 
-pub fn vid_init() -> VidScheme {
-    let srs = hotshot_types::data::test_srs(NUM_STORAGE_NODES);
-    VidScheme::new(NUM_CHUNKS, NUM_STORAGE_NODES, srs).unwrap()
+pub fn vid_init<TYPES: NodeType>(
+    membership: TYPES::Membership,
+    view_number: TYPES::Time,
+) -> VidScheme {
+    let num_committee = membership.get_committee(view_number).len();
+
+    // calculate the last power of two
+    // TODO change after https://github.com/EspressoSystems/jellyfish/issues/339
+    // issue: https://github.com/EspressoSystems/HotShot/issues/2152
+    let chunk_size = num_committee.next_power_of_two() / 2;
```

Comment:
I think I'm sort of wrong about this being the exact same as prev power of two.  This could overflow if `num_committee` is greater than greattest power of 2 `usize` can hold

Reply:
Good catch! I think we could use [checked_next_power_of_two](https://doc.rust-lang.org/std/primitive.usize.html#method.checked_next_power_of_two) then.

Reply:
If it returns `Some(power)`, then we just use `power / 2`. If it's `None`, then we know there's an overflow and we use the largest power of 2 of `usize`.

Reply:
Yeah we should be well under the limit at this point, if we get to 2^31 nodes this won't be the first to break.  I'm just worried that if we put it in the code and forget to come back to it and hit it on some weird system idk.

Reply:
> Yeah we should be well under the limit at this point, if we get to 2^31 nodes this won't be the first to break. I'm just worried that if we put it in the code and forget to come back to it and hit it on some weird system idk.

Alright, I'm going to do the checked variant. Should get us another factor of 2 in the worst case

Reply:
Sorry I missed this party. You could also do
```rust
let chunk_size = 1 << num_committee.ilog2()
```
[playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=99708dc39b2ee8c15a39434f71beb1bb)